code/game/objects/items/stunbaton.dm

```diff
@@ -193,13 +193,8 @@
 
 /// After the initial stun period, we check to see if the target needs to have the stun applied.
 /obj/item/melee/baton/proc/apply_stun_effect_end(mob/living/target)

```

Comment:
Is this necessary to leave in, or can I remove this bit entirely?

Reply:
yes, needed for delay